src/common/options/crimson.yaml.in

```diff
@@ -31,6 +31,15 @@ options:
   desc: CPU cores on which alienstore threads will run in cpuset(7) format
   flags:
   - startup
+- name: crimson_seastar_num_threads
+  type: uint
+  level: advanced
+  default: 0
+  desc: The number of threads for serving seastar reactors
```

Comment:
Can be more informative:

The number of threads for serving seastar reactors without CPU pinning, won't be effective if crimson_seastar_cpu_cores is set.